.github/workflows/aws-eks-test.yml

```diff
@@ -74,22 +79,27 @@ jobs:
 
       - name: create run artifacts
         if: always()
-        # Add any artifacts that should be assosciated with this run to /tmp/artifacts/${{ github.run_id }}
+        # Add any artifacts that should be associated with this run to /tmp/artifacts/${{ github.run_id }}
         run: |
           mkdir -p /tmp/artifacts/${{ github.run_id }} ||
-          kubectl logs -n acorn-system -l app=acorn-api > /tmp/artifacts/${{ github.run_id }}/acorn-api.log ||
-          kubectl logs -n acorn-system -l app=acorn-controller > /tmp/artifacts/${{ github.run_id }}/acorn-controller.log ||
+          kubectl logs -n acorn-system -l app=acorn-api > /tmp/artifacts/${{ github.run_id }}/logs/acorn-api.log ||
```

Comment:
Oh nice catch! Yes, I this the actual issue. The intent here was to make it so that the job didn't fail out if one step failed but it was making it so none of the other commands where running.